source/install/prod-docker.rst

```diff
@@ -23,7 +23,14 @@ Production Docker Setup on Ubuntu
        sudo service docker start
        newgrp docker
 
-2. **Deploy the Mattermost Production Docker** setup by running:
+2. **Install Docker Compose** using `the online guide <https://docs.docker.com/compose/install/>`_ or these instructions:
+
+   .. code:: bash
+   
+      curl -L https://github.com/docker/compose/releases/download/1.14.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
This version is already out of date. Maybe we should point to the download page?